linera-core/src/node.rs

```diff
@@ -45,9 +45,9 @@ pub trait ValidatorNode {
     ) -> Result<ChainInfoResponse, NodeError>;
 
     /// Processes a certificate without a value.
-    async fn handle_lite_certificate(
+    async fn handle_lite_certificate<'a>(
```

Comment:
Nit: I believe all lifetimes in these methods can be ellided, so the `<'a>` generic lifetime can be removed, and the type can be just `LiteCertificate<'_>`.